Docker/entrypoint.sh

```diff
@@ -23,4 +20,50 @@ if [ -n "$CRON_MIN" ]; then
 	crontab -l | sed -r "\\#FreshRSS#s#^[^ ]+ #$CRON_MIN #" | crontab -
 fi
 
+if [ -n "$FRESHRSS_INSTALL" ]; then
+	php -f ./cli/do-install.php -- \
+		$(echo "$FRESHRSS_INSTALL" | sed -r 's/[\r\n]+/\n/g' | paste -s -) \
```

Comment:
```suggestion
		"$(echo "$FRESHRSS_INSTALL" | sed -r 's/[\r\n]+/\n/g' | paste -s -)" \
```

Reply:
Hum, that does not work. We do have multiple options, not a single one ;-)
With the double quotes, it is transformed into a single parameter

Reply:
What we receive is something like that:
```
--api_enabled --base_url https://rss.example.net --db-base freshrss --db-host freshrss-db --db-password freshrss --db-type pgsql --db-user freshrss --default_user admin --language en
```

Reply:
Are we being POSIX compliant? Otherwise we could switch the shebang to /bin/bash to make it clear we want word splitting and that it's not an oversight. The other alternative is a `shellcheck disable=SC1234`.
```
read -ra args <<(echo "$FRESHRSS_INSTALL" | sed -r 's/[\r\n]+/\n/g' | paste -s -)
php "${args[@]}"
```

I double checked btw, and shellcheck didn't complain about anything worse.

Reply:
I was trying to stick to POSIX, yes. At the very least, must work with the default shell in Debian and Alpine, which are the ones which will be executing that for now

Reply:
So what do you suggest, @Frenzie ?

Reply:
Either shellcheck disable so we know it's on purpose or just nothing. :-)